docs/greenboot_dev.md

```diff
@@ -41,6 +41,9 @@ FAILURE
 ...
 ```
 
+> Run the `sudo journalctl -o cat -u redboot-task-runner.service` command
```

Comment:
where is the `redboot-task-runner.service` added?

Reply:
This service is added by Greenboot when its RPM is installed.